src/qml/qml.pro

```diff
@@ -84,7 +84,9 @@ QMAKE_EXTRA_TARGETS += qmltypes
 TS_FILE = $$OUT_PWD/socialcache.ts
 EE_QM = $$OUT_PWD/socialcache_eng_en.qm
 
-ts.commands += lupdate $$PWD/.. -ts $$TS_FILE
+qtPrepareTool(LUPDATE, lupdate)
```

Comment:
Hm, why is this needed? Don't recall seeing too much usage of qtPrepareTool, not finding documentation and so far it's been working ok just telling qmake what commands to execute.

Reply:
the issue is that some OS provides lupdate-qt5, some lupdate-qt6, and others just lupdate. Also, the lupdate can be in different path e.g. /opt/qt5

Reply:
Ah right. Fair enough then, if the call handles varying binary names and locations.